### PR TITLE
config/lenovo-a6010: add wingtech,qimei-ili9881 panel driver 

### LINKED ISSUE #18: [Panel] Lenovo A6000's panel qimei isn't support yet
I was install pmos via pmbootstrap and images from official website. I thought i can install and boot normally (with booting display screen) but it wasn't.
The screen goes fade away (and does nothing?)
I post this issue for footage.

### PR DESCRIPTION
**Pull request by Mas4hmad**
add panel for lenovo-a6010

Fix #18

### CODE REVIEW

File: config/lenovo-a6010.dumb.sh [hunk 1]
```diff
@@ -4,4 +4,6 @@
 OPTIONS=(-r vsp -r vsn --dumb-dcs)
 PANELS=(
 	[ili9881_720p_video]="wingtech,yassy-ili9881"
+	[ili9881_qimei_720p_video]="wingtech,qimei-ili9881"
+
```
Comment (stephan-gh) [inline config/lenovo-a6010.dumb.sh:8]: Drop this line please

**Comment by stephan-gh** [comment]
Can you confirm that the panel works correctly now? In #18 you mentioned that it does not work correctly "black-faded screen" etc.

**Comment by Mas4hmad** [comment]
> Can you confirm that the panel works correctly now? In #18 you mentioned that it does not work correctly "black-faded screen" etc.

I've dropped the link below. You'll see my device can display at 9 seconds at last video
https://youtube.com/shorts/km-xEVlRTEg?feature=share

**Comment by Mas4hmad** [comment]
Ready for review

**Review by stephan-gh — APPROVED** [review]

**Comment by Mas4hmad** [comment]
will this PR to be merged? i tought i was could report issue#18 is fixed and create PR

**Pull request merged**